src/less/skeleton-loader/skeleton-loader.less

```diff
@@ -0,0 +1,224 @@
+@import "../variables/variables.less";
+
+@skeleton-text-border-radius: 3px;
```

Comment:
Especially because we are doing this granular approach for skeleton loaders where the teams have to build them themselves, I think it's especially important that teams have full autonomy to control all of these details so they can match exactly with their designs. Therefore, I think I'd prefer to see these as CSS variables instead of LESS variables.

Reply:
Yeah, agree with you here. Contemplated for a while on that. But used less for now because of consistency reasons. During less to css conversion, we discussed that all the variables internal to the component need to be in less for better DX. If and when we move to CSS vars, I definitely prefer css vars.

Reply:
The one's that were defined in less are DS specific and all the other generalized ones are defined in css variables.